Testing/MqttTests.cs

```diff
@@ -15,6 +16,8 @@ public class MqttTests
     {
         private string iotHubConnectionString;
         private string iotHubDeviceConnectionString;
+        private string iotHubDeviceCertifConnectionString;
```

Comment:
Nitpicking here :) but can we use either "Certificate" or "Cert" in the variable names?